services/sagemaker/generator.yaml

```diff
@@ -1,10 +1,41 @@
+operations:
+  StopTrainingJob:
+    operation_type: Delete
+    resource_name: TrainingJob
 resources:
   Model:
     exceptions:
       errors:
         404:
           code: ValidationException
           message_prefix: Could not find model
+  TrainingJob:
+    exceptions:
+      errors:
+          404:
+            code: ValidationException
+            message_prefix: Requested resource not found
+    fields:
+      TrainingJobStatus:
+          is_read_only: true
```

Comment:
Add `is_printable` for `TrainingJobStatus`, `SecondaryStatus` and `FailureReason`

Reply:
oh yea, I noticed that was missing. Let me create a separate Pr for that.

Reply:
what do these mean, is_read_only, is_printable (probably printer columns)